### DISCUSSION TITLE
cs/tcs/toc/topic4/

### DISCUSSION THREAD

**Discussion by giscus** [TonyCrane/note#43] [General]
# cs/tcs/toc/topic4/

鹤翔万里（TonyCrane）的笔记本

https://note.tonycrane.cc/cs/tcs/toc/topic4/

**Comment by birchtree2** [comment]
太强了。

话说 ![](https://note.tonycrane.cc/assets/images/cs/tcs/toc/topic4/img1.dark.png#only-dark)

这图semidecide和decide是不是写反了
